proto/marshal/compat.py

```diff
@@ -19,6 +19,9 @@
 # not be included.
 
 from google.protobuf.internal import containers
+import google.protobuf
+
+PROTOBUF_VERSION = google.protobuf.__version__
```

Comment:
Not a blocker, but just a reminder that we could have a couple of lines here to make the version into a tuple of mostly numbers, as we've talked about in a different context. That would get rid of string comparisons and list membership checks below.

Not a biggie; we can defer until we do this consistently across the various repos.